File: app/src/main/java/com/moon/beautygirlkotlin/network/api/GankMeiziApi.java
package com.moon.beautygirlkotlin.network.api;



import com.moon.beautygirlkotlin.mengmeizi.model.GankMeiziResult;

import retrofit2.http.GET;
import retrofit2.http.Path;
import rx.Observable;

public interface GankMeiziApi {

  /**
   * gank妹子,福利
   */
  @GET("data/福利/{number}/{page}")
  Observable<GankMeiziResult> getGankMeizi(@Path("number") int number, @Path("page") int page);
}

File: app/src/main/java/com/moon/beautygirlkotlin/network/api/GankMeiziAPI.kt
package com.moon.beautygirlkotlin.network.api

import com.moon.beautygirlkotlin.gank.model.GankMeiziResult
import retrofit2.http.GET
import retrofit2.http.Path

/**
 * author: moon
 * created on: 18/5/3 下午5:01
 * description:
 */
interface GankMeiziAPI {

    @GET("data/福利/{number}/{page}")
    suspend fun getGankMeizi(@Path("number") number: Int, @Path("page") page: Int): GankMeiziResult
}